ci-operator/step-registry/ipi/deprovision/deprovision/ipi-deprovision-deprovision-commands.sh

```diff
@@ -4,11 +4,21 @@ set -o nounset
 set -o errexit
 set -o pipefail
 
+trap 'CHILDREN=$(jobs -p); if test -n "${CHILDREN}"; then kill ${CHILDREN} && wait; fi' TERM
+
 cluster_profile=/var/run/secrets/ci.openshift.io/cluster-profile
 export AWS_SHARED_CREDENTIALS_FILE=$cluster_profile/.awscred
 export AZURE_AUTH_LOCATION=$cluster_profile/osServicePrincipal.json
 
 echo "Deprovisioning cluster ..."
 cp -ar "${SHARED_DIR}" /tmp/installer
-openshift-install --dir /tmp/installer destroy cluster
-cp /tmp/installer/.openshift_install.log "${ARTIFACT_DIR}/"
+openshift-install --dir /tmp/installer destroy cluster &
+
+set +e
+wait "$!"
+ret="$?"
+set -e
+
+cp /tmp/installer/.openshift_install.log "${ARTIFACT_DIR}"
+
+exit "$ret"
```

Comment:
I was thinking about this the other day, would creating symlinks before executing the installer work in these cases? We could then drop this type of logic from this and other steps.

Reply:
> I was thinking about this the other day, would creating symlinks before executing the installer work in these cases?

Might work for `destroy cluster`, but would not work for install because we need to `grep` or `sed` to drop the kubeadmin password.  I'll try if I can sort out the unbounded-variable thing...